src/index.scss

```diff
@@ -471,6 +466,11 @@ header {
   width: 24px !important;
 }
 
+.spinner-dimensions-36 {
+  height: 36px !important;
+  width: 36px !important;
+}
+
```

Comment:
No need to change the spinner dimensions.

Reply:
The "spinner-dimensions" class was previously utilized with dimensions specified as 24px for both height and width, whereas the Figma design indicates dimensions of 36px.

Reply:
We are using a spinner- dimensions 24px all over the app. Please revert this change to 24px to make it consistent.